### DISCUSSION TITLE
how to re-edit Strategy

### DISCUSSION THREAD

**Discussion by StephenPan** [snapshot-labs/snapshot#1085] [Q&A] [answered]
There is a question, after creating a Spaces, how to modify the parameters, for example, I use the default Dai in Erc20-Balance-Of, how to re-edit Strategy after submitting, and will vote for the voting condition snapshotDAI changed to the token I want?

**Comment by metrox-eth** [comment, marked as answer]
When you edit strategy parameters, you'll need to duplicate / create a new proposal for the change to take effects
